adabot_description/urdf/adabot.macros.xacro

```diff
@@ -36,30 +12,43 @@
 
         <!-- The name of the current wheel -->
         <xacro:property name="wheel_name" value="${suffix}_wheel" />
+        
+        <!-- Create the wheel geometry (for visual debug and collisions) -->
+        <xacro:property name="wheel_geometry">
+            <!-- Cylinders default with their height along the z axis -->
+            <origin xyz="0 0 0" rpy="${PI/2} 0 0" />
+            <geometry>
+                <!-- <cylinder length="${wh_length}" radius="${wh_radius}" /> -->
```

Comment:
Cylinders for wheels can be removed. We should have two options: spheres and STL meshes.